test/config/config.js

```diff
@@ -28,6 +28,11 @@ module.exports = {
     password: process.env.SEQ_MSSQL_PW   || process.env.SEQ_PW   || null,
     host:     process.env.SEQ_MSSQL_HOST || process.env.SEQ_HOST || '127.0.0.1',
     port:     process.env.SEQ_MSSQL_PORT || process.env.SEQ_PORT || 11433,
+    dialectOptions: {
+      instanceName:  process.env.MSSQL_INSTANCE ? process.env.MSSQL_INSTANCE : 'SQLEXPRESS',
+      // big insert queries need a while
```

Comment:
@harshithkashyap There are a few cases of connections which seem quite tricky. I found that I did need the instanceName to connect to my SQL Express even with the host and port - so I think that we do need a way of putting in the dialectOptions - but I can see that you're right that it's not good to default it as that's a breaking change.

Can you check to see if it works for you if you set MSSQL_INSTANCE to blank?

If that works it would be easiest just to make that the default and then people like me who need it can just set it to SQLEXPRESS. Otherwise I'll have to make the whole dialectoptions conditional on the existence of instance.